apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts

```diff
@@ -81,6 +89,10 @@ export class VoMembersComponent implements OnInit {
     this.route.parent.params.subscribe(parentParams => {
       const voId = parentParams['voId'];
 
+      this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
+        this.blockManualMemberAdding = attrValue.value !== null;
+      });
+
```

Comment:
I am really sorry, I think I didnt explain properly what I wanted. What I wanted is the code below. 
basically inserting the asynchrounous operation into the subscrible block - the reason is that we want to wait for end of one asynchronous operation and then start another, so we really know when we can actually load the page.
Please the same do in the group-members component

      this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
        this.blockManualMemberAdding = attrValue.value !== null;

        this.voService.getVoById(voId).subscribe(vo => {
        this.vo = vo;
        this.setAuthRights();
        this.memberMethodService.getMembersCount(this.vo.id).subscribe( count => {
          this.count = count;
          if(count < 400) {
            this.onListAll();
          }
          this.loading = false;
        });
      });

Reply:
Sorry, I didn't figure it out correctly. This implemetation should be better.